src/main/java/competition/subsystems/drive/commands/AlignWithCreeperCommandWCalc.java

```diff
@@ -0,0 +1,79 @@
+package competition.subsystems.drive.commands;
+
+import competition.electrical_contract.ElectricalContract;
+import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.drive.logic.AlignWithCreeperCalculator;
+import competition.subsystems.oracle.ReefCoordinateGenerator;
+import competition.subsystems.pose.Cameras;
+import competition.subsystems.pose.PoseSubsystem;
+import competition.subsystems.vision.AprilTagVisionSubsystemExtended;
+import competition.subsystems.vision.CoprocessorCommunicationSubsystem;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import org.kobe.xbot.JClient.CachedSubscriber;
+import org.kobe.xbot.JClient.XTablesClient;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.PIDManager;
+import xbot.common.math.PIDManager.PIDManagerFactory;
+import xbot.common.math.XYPair;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+import xbot.common.properties.StringProperty;
+import xbot.common.subsystems.drive.control_logic.HeadingModule;
+import competition.subsystems.drive.logic.AlignWithCreeperCalculator;
+import javax.inject.Inject;
+
+
+public class AlignWithCreeperCommandWCalc extends BaseCommand {
```

Comment:
⭐ ⭐ Would recommend Command being the suffix here to match our other commands
```suggestion
public class AlignWithCreeperWCalcCommand extends BaseCommand {

Reply:
👍